### ISSUE TITLE
admins not working

### ISSUE THREAD

**Issue by ALegitCookie** [roflmuffin/CounterStrikeSharp#332]
hello, i've got my admins.json (in plugins/counterstrikesharp/conifgs) configured in the following fashion:

`{
  "wizard": {
    "identity": "STEAM_0:0:169877904",
    "flags": ["@css/changemap", "@css/generic"]
  }
}`

but when i do !css in chat, it says i do not have the permission @css/generic.
what can i do to make it work?
thanks

**Comment by roflmuffin** [comment]
Please run the command `css_admins_list` and verify if your user is displayed. Also please ensure that the example config has had the `.example` suffix removed from the filename.

**Comment by ALegitCookie** [comment]
> Please run the command `css_admins_list` and verify if your user is displayed. Also please ensure that the example config has had the `.example` suffix removed from the filename.

![image](https://github.com/roflmuffin/CounterStrikeSharp/assets/39338269/63a46235-3573-42b2-8f46-14fac46f2203)

i can confirm my steamid shows up and you can see the flags when running the command provided. however, it still insists in-game that i do not have access to the commands.

**Comment by roflmuffin** [comment]
Its possible that your server is failing to authorise the users steam ID, are there any logs present about steam ID authorisation when you connect to the server?

**Comment by ALegitCookie** [comment]
> Its possible that your server is failing to authorise the users steam ID, are there any logs present about steam ID authorisation when you connect to the server?

yes i can see that the server can see the user's steamID when they join, pictured.

i also can see when starting up that CSSharp says in console that it read 1 admin in the admin file. (not pictured but i read it myself and confirmed)

![image](https://github.com/roflmuffin/CounterStrikeSharp/assets/39338269/49979915-5ab6-4092-989d-4bfab290b841)

**Comment by roflmuffin** [comment]
Do you have any overrides configured in the `admin_overrides.json` file? And also can you provide which version of CS# you are running?

**Comment by ALegitCookie** [comment]
> Do you have any overrides configured in the `admin_overrides.json` file? And also can you provide which version of CS# you are running?

haven't touched the admin overrides file, i'm on most recent version

**Comment by Koudkun** [comment]
I did it like this:

```
{
  "Koud": {
    "identity": "76561198032257636",
    "flags": ["@css/generic", "@css/kick", "@css/ban", "@css/unban", "@css/slay", "@css/changemap", "@css/chat", "@css/vote"],
    "groups": ["#css/admin"]
  },
  "Niklas": {
    "identity": "76561198396446262",
    "flags": ["@css/generic", "@css/kick", "@css/ban", "@css/unban", "@css/slay", "@css/changemap", "@css/chat", "@css/vote"],
    "groups": ["#css/admin"]
  }
}
```
For me it is working fine.
I just use core.json and admins.json
You can leave the rest as an example (override and groups)
